diff --git a/themeAugmentation/components.d.ts b/themeAugmentation/components.d.ts
index 7ec1df957c5555c872ef4630fd36ba1de674b2b1..ff9f15d42fd25fcd5566f30728a06382ebba268c 100644
--- a/themeAugmentation/components.d.ts
+++ b/themeAugmentation/components.d.ts
@@ -195,6 +195,6 @@ export interface PickerComponents<Theme = unknown> {
   };
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/components' {
   interface Components<Theme = unknown> extends PickerComponents<Theme> {}
 }
diff --git a/themeAugmentation/overrides.d.ts b/themeAugmentation/overrides.d.ts
index 0a845aa4d33d51824675323b2891c4cc0607e400..dd1a404d72212690343daca0f716e8e4b53440c9 100644
--- a/themeAugmentation/overrides.d.ts
+++ b/themeAugmentation/overrides.d.ts
@@ -78,7 +78,7 @@ export interface PickersComponentNameToClassKey {
   MuiPickersSectionList: PickersSectionListClassKey
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/overrides' {
   interface ComponentNameToClassKey extends PickersComponentNameToClassKey {}
 }
 
diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index c8fc7f3949c2a52b297848ed8659ade1f00e04bf..ed3c5dbfeeddbc33db37ad74c25fb1d91b5762d6 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -114,7 +114,7 @@ export interface PickersComponentsPropsList {
   MuiPickersSectionList: PickersSectionListProps;
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
   interface ComponentsPropsList extends PickersComponentsPropsList {}
 }
 
