diff --git a/themeAugmentation/components.d.ts b/themeAugmentation/components.d.ts
index 8dbd4cff0aa8b24fc749192eae8bf578d2b0e252..8b14bedd517c4dce5340c228779055a0d04cf758 100644
--- a/themeAugmentation/components.d.ts
+++ b/themeAugmentation/components.d.ts
@@ -28,6 +28,6 @@ export interface TreeViewComponents<Theme = unknown> {
   };
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/components' {
   interface Components<Theme = unknown> extends TreeViewComponents<Theme> {}
 }
diff --git a/themeAugmentation/overrides.d.ts b/themeAugmentation/overrides.d.ts
index 6402fef94e92277a5656960ab98701ee99f4063b..74f8399c53edb7ee958a111915ffa20ba22fb150 100644
--- a/themeAugmentation/overrides.d.ts
+++ b/themeAugmentation/overrides.d.ts
@@ -12,7 +12,7 @@ export interface TreeViewComponentNameToClassKey {
   MuiTreeItem2: TreeItemClassKey;
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/overrides' {
   interface ComponentNameToClassKey extends TreeViewComponentNameToClassKey {}
 }
 
diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index da5ae41f5bd464117537244433f3a58ebf909f14..7240d9a586b94c03f9111f73f8a0f1c12da12850 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -12,7 +12,7 @@ export interface TreeViewComponentsPropsList {
   MuiTreeItem2: TreeItem2Props;
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
   interface ComponentsPropsList extends TreeViewComponentsPropsList {}
 }
 
