diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index f7dab1065b1436f2adccbbf7e96a4fcd1c3d0116..27d43171bcab5daac9a31ae9093bc4368b15a14a 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -1,4 +1,4 @@
-import { ComponentsOverrides, ComponentsProps } from '@mui/material/styles';
+import { ComponentsOverrides, ComponentsProps, Theme } from '@mui/material/styles';
 import { DataGridProProps } from '../models/dataGridProProps';
 export interface DataGridProComponentsPropsList {
     MuiDataGrid: DataGridProProps;
@@ -6,10 +6,10 @@ export interface DataGridProComponentsPropsList {
 export interface DataGridProComponents {
     MuiDataGrid?: {
         defaultProps?: ComponentsProps['MuiDataGrid'];
-        styleOverrides?: ComponentsOverrides['MuiDataGrid'];
+        styleOverrides?: ComponentsOverrides<Theme>['MuiDataGrid'];
     };
 }
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
     interface ComponentsPropsList extends DataGridProComponentsPropsList {
     }
 }
