diff --git a/themeAugmentation/overrides.d.ts b/themeAugmentation/overrides.d.ts
index 577d78333a4afe2bfb9413eb9071cdf037ad83cd..d8d49ee860efc21b61c60171bc650f8730e883d7 100644
--- a/themeAugmentation/overrides.d.ts
+++ b/themeAugmentation/overrides.d.ts
@@ -2,7 +2,7 @@ import { GridClassKey } from '../constants/gridClasses';
 export interface DataGridComponentNameToClassKey {
     MuiDataGrid: GridClassKey;
 }
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/overrides' {
     interface ComponentNameToClassKey extends DataGridComponentNameToClassKey {
     }
     interface Mixins {
diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index 83c835c9ce370d4aafc82f19f5f84eb32143d61a..f395cc75a647ee154e011ac3819cafc23d769187 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -9,7 +9,7 @@ export interface DataGridComponents<Theme = unknown> {
         styleOverrides?: ComponentsOverrides<Theme>['MuiDataGrid'];
     };
 }
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
     interface ComponentsPropsList extends DataGridComponentsPropsList {
     }
     interface Components<Theme = unknown> extends DataGridComponents<Theme> {
