diff --git a/themeAugmentation/components.d.ts b/themeAugmentation/components.d.ts
index 881d1d066a3541043076cc181eada64cefb3a6ad..d037133e8e8f37c70d900ce8ead40112e81103fa 100644
--- a/themeAugmentation/components.d.ts
+++ b/themeAugmentation/components.d.ts
@@ -76,6 +76,6 @@ export interface PickersProComponents<Theme = unknown> {
   };
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/components' {
   interface Components<Theme = unknown> extends PickersProComponents<Theme> {}
 }
diff --git a/themeAugmentation/overrides.d.ts b/themeAugmentation/overrides.d.ts
index 8b53b859f7c33cb5eda1cad86f4e0ff7e1671736..9b98f51ec117ac7589c768bcddbb5f06ea647218 100644
--- a/themeAugmentation/overrides.d.ts
+++ b/themeAugmentation/overrides.d.ts
@@ -22,7 +22,7 @@ export interface PickersProComponentNameToClassKey {
   MuiMultiInputTimeRangeField: MultiInputRangeFieldClassKey;
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/overrides' {
   interface ComponentNameToClassKey extends PickersProComponentNameToClassKey {}
 }
 
diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index 6536f000c4944cbd202c6e61dea0904c034e6fca..61324ea7a29f5a3dc02c89a788676e6a33483bae 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -49,7 +49,7 @@ export interface PickersProComponentsPropsList {
   MuiMobileDateTimeRangePicker: MobileDateTimeRangePickerProps<PickerValidDate, any>;
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
   interface ComponentsPropsList extends PickersProComponentsPropsList {}
 }
 
