diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index f428bae2fef2a64d17b300d0277be3f284f6e3f3..27d43171bcab5daac9a31ae9093bc4368b15a14a 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -9,7 +9,7 @@ export interface DataGridProComponents {
         styleOverrides?: ComponentsOverrides<Theme>['MuiDataGrid'];
     };
 }
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
     interface ComponentsPropsList extends DataGridProComponentsPropsList {
     }
 }
