diff --git a/themeAugmentation/components.d.ts b/themeAugmentation/components.d.ts
index 9971dbc94baf3fbe7c24782fcf6bdfe7704d8e34..1a6dc6c031a4c45f7dd8c1a51f051ff0f8d36f08 100644
--- a/themeAugmentation/components.d.ts
+++ b/themeAugmentation/components.d.ts
@@ -1,13 +1,13 @@
 import { ComponentsProps, ComponentsOverrides, ComponentsVariants } from '@mui/material/styles';
 
-export interface PickersProComponents {
+export interface PickersProComponents<Theme = unknown> {
   MuiDateRangePickerDay?: {
     defaultProps?: ComponentsProps['MuiDateRangePickerDay'];
-    styleOverrides?: ComponentsOverrides['MuiDateRangePickerDay'];
+    styleOverrides?: ComponentsOverrides<Theme>['MuiDateRangePickerDay'];
     variants?: ComponentsVariants['MuiDateRangePickerDay'];
   };
 }
 
-declare module '@mui/material/styles' {
-  interface Components extends PickersProComponents {}
+declare module '@mui/material/styles/components' {
+  interface Components<Theme = unknown> extends PickersProComponents<Theme> {}
 }
diff --git a/themeAugmentation/overrides.d.ts b/themeAugmentation/overrides.d.ts
index 3868c53ba5227a1c4fe7a7b238f0e64ddeb6aecb..36063d0e217d19948ae336f8760eb81fe6b55ce3 100644
--- a/themeAugmentation/overrides.d.ts
+++ b/themeAugmentation/overrides.d.ts
@@ -5,7 +5,7 @@ export interface PickersProComponentNameToClassKey {
   MuiDateRangePickerDay: DateRangePickerDayClassKey;
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/overrides' {
   interface ComponentNameToClassKey extends PickersProComponentNameToClassKey {}
 }
 
diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index 5e60e0b9e588599c3822f8b64a5f16189f561c8c..f7d2f6d74ed9ede173eff50c4c8e4355f2f6cc2e 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -4,7 +4,7 @@ export interface PickersProComponentsPropsList {
   MuiDateRangePickerDay: DateRangePickerDayProps<unknown>;
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
   interface ComponentsPropsList extends PickersProComponentsPropsList {}
 }
 
