diff --git a/themeAugmentation/components.d.ts b/themeAugmentation/components.d.ts
index 20c908847f6efaeb8308816c744bd850301a82de..12727942b33284aa8b90c06b98a8b2227d54b2b8 100644
--- a/themeAugmentation/components.d.ts
+++ b/themeAugmentation/components.d.ts
@@ -78,6 +78,6 @@ export interface LabComponents {
   };
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/components' {
   interface Components extends LabComponents {}
 }
diff --git a/themeAugmentation/overrides.d.ts b/themeAugmentation/overrides.d.ts
index bf8e7a6e718558e0bfbcd7bcb3873e8af3f25dec..aa73e3ed6ad621afb818833a3a39c7a083fa50e6 100644
--- a/themeAugmentation/overrides.d.ts
+++ b/themeAugmentation/overrides.d.ts
@@ -29,7 +29,7 @@ export interface LabComponentNameToClassKey {
   MuiTreeView: TreeViewClassKey;
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/overrides' {
   interface ComponentNameToClassKey extends LabComponentNameToClassKey {}
 }
 
diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index 0024906b9b8eb27f883612f29b84e92eae704bcb..b3eb2dfdbb4cacd0b9c9261a9957391a0ea71c62 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -30,7 +30,7 @@ export interface LabComponentsPropsList {
   MuiTreeView: TreeViewProps;
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
   interface ComponentsPropsList extends LabComponentsPropsList {}
 }
 
