diff --git a/themeAugmentation/overrides.d.ts b/themeAugmentation/overrides.d.ts
index 923764acbad4b11f11d196d628a8ba10e9e64699..fb9e2a854a6512d542a9d84d3c4401a46d48a95e 100644
--- a/themeAugmentation/overrides.d.ts
+++ b/themeAugmentation/overrides.d.ts
@@ -2,7 +2,7 @@ import { GridClassKey } from '../constants/gridClasses';
 export interface DataGridComponentNameToClassKey {
     MuiDataGrid: GridClassKey;
 }
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/overrides' {
     interface ComponentNameToClassKey extends DataGridComponentNameToClassKey {
     }
 }
diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index 83c835c9ce370d4aafc82f19f5f84eb32143d61a..7fce855cb2ca4fd94a27ca3b9785fe0bf6eef903 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -9,9 +9,11 @@ export interface DataGridComponents<Theme = unknown> {
         styleOverrides?: ComponentsOverrides<Theme>['MuiDataGrid'];
     };
 }
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
     interface ComponentsPropsList extends DataGridComponentsPropsList {
     }
+}
+declare module '@mui/material/styles/components' {
     interface Components<Theme = unknown> extends DataGridComponents<Theme> {
     }
 }
