diff --git a/themeAugmentation/components.d.ts b/themeAugmentation/components.d.ts
index 05fa0cf3d92ba06747599c0cea83e3949eb9fb7e..0045158fea463ce2beafadd38ff274e586a4ede1 100644
--- a/themeAugmentation/components.d.ts
+++ b/themeAugmentation/components.d.ts
@@ -73,6 +73,6 @@ export interface LabComponents {
   };
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/components' {
   interface Components extends LabComponents {}
 }
diff --git a/themeAugmentation/overrides.d.ts b/themeAugmentation/overrides.d.ts
index 422fda85f0b55076279456973318e2aacf858d1f..fd5a3ef006de47b99876bfb0ab11a3f9a38a0b2e 100644
--- a/themeAugmentation/overrides.d.ts
+++ b/themeAugmentation/overrides.d.ts
@@ -27,7 +27,7 @@ export interface LabComponentNameToClassKey {
   MuiTreeView: TreeViewClassKey;
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/overrides' {
   interface ComponentNameToClassKey extends LabComponentNameToClassKey {}
 }
 
diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index c8fcc683b3dda7971bcf64f17565c5dbada33493..92b9d74d49e75bd3c5f2d441a19ee8aac1da4701 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -28,7 +28,7 @@ export interface LabComponentsPropsList {
   MuiTreeView: TreeViewProps;
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
   interface ComponentsPropsList extends LabComponentsPropsList {}
 }
 
