diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index 1eb19ca8c5b026e36ae146351f298b253b22f3a0..ce15a94dc4da7b1873d6c6aa9f77dde98843f5e8 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -9,9 +9,11 @@ export interface DataGridPremiumComponents<Theme = unknown> {
         styleOverrides?: ComponentsOverrides<Theme>['MuiDataGrid'];
     };
 }
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
     interface ComponentsPropsList extends DataGridPremiumComponentsPropsList {
     }
+}
+declare module '@mui/material/styles/components' {
     interface Components<Theme = unknown> extends DataGridPremiumComponents<Theme> {
     }
 }
