diff --git a/themeAugmentation/components.d.ts b/themeAugmentation/components.d.ts
index 9827b5f8e8abdd0eef2d61440e07d82769c5f73d..1511ea17280cc5229c87658dcdc2aca85ece5032 100644
--- a/themeAugmentation/components.d.ts
+++ b/themeAugmentation/components.d.ts
@@ -48,6 +48,6 @@ export interface PickerComponents<Theme = unknown> {
   };
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/components' {
   interface Components<Theme = unknown> extends PickerComponents<Theme> {}
 }
diff --git a/themeAugmentation/overrides.d.ts b/themeAugmentation/overrides.d.ts
index 4383e1b810bba3ab1adfea386ea66985c9386fc1..f401d57f1d28db8fecbdbbae2067b3ddd5aad905 100644
--- a/themeAugmentation/overrides.d.ts
+++ b/themeAugmentation/overrides.d.ts
@@ -19,7 +19,7 @@ export interface PickersComponentNameToClassKey {
   MuiPickerStaticWrapper: PickerStaticWrapperClassKey;
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/overrides' {
   interface ComponentNameToClassKey extends PickersComponentNameToClassKey {}
 }
 
diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index 6a3fbd371fc122319730d7005c5732ad91e87ee4..66988b5f9ede7113fe25e66b71a9c5f3a54c3b74 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -38,7 +38,7 @@ export interface PickersComponentsPropsList {
   MuiPickerStaticWrapper: PickerStaticWrapperProps;
 }
 
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
   interface ComponentsPropsList extends PickersComponentsPropsList {}
 }
 
