diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index 5096ea8dcea6ce2d93a3ac68fe206a9a1acbdab1..023877f3a8ae83b67c9963c74b08cb1b87773a9b 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -9,7 +9,7 @@ export interface DataGridPremiumComponents {
         styleOverrides?: ComponentsOverrides<Theme>['MuiDataGrid'];
     };
 }
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
     interface ComponentsPropsList extends DataGridPremiumComponentsPropsList {
     }
 }
