diff --git a/themeAugmentation/props.d.ts b/themeAugmentation/props.d.ts
index cab8e4edc5971fa67829389b17ef3dd67c48c0ae..5d0db1dea1e900f924ff4bb918fcb7cd1afcb8bf 100644
--- a/themeAugmentation/props.d.ts
+++ b/themeAugmentation/props.d.ts
@@ -9,7 +9,7 @@ export interface DataGridComponents {
         styleOverrides?: ComponentsOverrides<Theme>['MuiDataGrid'];
     };
 }
-declare module '@mui/material/styles' {
+declare module '@mui/material/styles/props' {
     interface ComponentsPropsList extends DataGridComponentsPropsList {
     }
 }
